behave/features/steps/common_steps.py

```diff
@@ -75,6 +74,11 @@ def step_click_continue(context):
         pass
 
 
+@step(u'I click continue')
+def step_click_continue(context):
+    step_click_submit(context)
```

Comment:
We have been doing `context.execute_steps` to execute other steps from a step. What's the difference between how you are doing it and the `context.execute_steps`?